### PR TITLE
Bump version to v6.0.5

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
New ABI version: 13.0.5

### CODE REVIEW

File: rpm/spec.in [hunk 1]
```diff
@@ -93,6 +93,13 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/%{name}/libdrizzle-redux/visibility.h
 
 %changelog
+* Tue Jun 20 2017 Andreas Bok Andersen <andreas.bok@sociomantic.com> - 6.0.5
+- Bug: Fix errors in rpm/spec.in file (2017-06-20)
```
Comment (daniel-zullo) [inline rpm/spec.in]: I think it should be "fix" (without capital letter). Otherwise the line below is wrong

**Review by daniel-zullo — COMMENTED** [review]
Apart from my cosmetic comment, LGTM

**Comment by andreas-bok-sociomantic** [comment]
Updated `rpm/spec.in`

**Comment by daniel-zullo** [comment]
Are you sure you've pushed the change?

**Pull request merged**